client/my-sites/menus/menu.jsx

```diff
@@ -228,6 +226,7 @@ var Menu = React.createClass( {
 						setEditItem={ this.setEditItem }
 						editedItem={ this.getEditItem() }
 						moveState={ this.state.moveState }
+						dragState={ this.state.draggedItem }
```

Comment:
Would it make sense to call the new prop `draggedItem` instead of `dragState`, since it contains no state other than the item?